reactor3/src/test/java/com/linecorp/armeria/common/reactor3/ContextAwareFluxTest.java

```diff
@@ -370,23 +368,6 @@ void connectableFlux() {
                     .verifyComplete();
     }
 
-    @Test
-    void connectableFlux_dispose() {
-        final ClientRequestContext ctx = newContext();
-        final Flux<String> flux;
-        final CompletableFuture<Disposable> future = new CompletableFuture<>();
-        try (SafeCloseable ignored = ctx.push()) {
-            final ConnectableFlux<String> connectableFlux = Flux.just("foo").publish();
-            flux = addCallbacks(connectableFlux.autoConnect(2, disposable -> {
-                assertThat(ctxExists(ctx)).isTrue();
-                future.complete(disposable);
-            }).publishOn(Schedulers.single()), ctx);
-        }
-        flux.subscribe().dispose();
-        flux.subscribe().dispose();
-        assertThat(future.join().isDisposed()).isTrue();
```

Comment:
`isDisposed()` doesn't return `true` anymore after the rework of `FluxPublisher`: https://github.com/reactor/reactor-core/pull/3538/files
The Javadoc says that returning `true` isn't guaranteed so I just removed this test case. 
```java
* Optionally return {@literal true} when the resource or task is disposed.
* <p>
* Implementations are not required to track disposition and as such may never
* return {@literal true} even when disposed.
```